features/step_definitions/wallets-steps.js

```diff
@@ -401,6 +401,11 @@ Then(/^I should be on the "([^"]*)" wallet "([^"]*)" screen$/, async function (w
   return waitUntilUrlEquals.call(this, `/wallets/${wallet.id}/${screenName}`);
 });
 
+Then(/^I should be on the "([^"]*)" screen$/, async function (screenName) {
+  console.log('screenName', screenName);
```

Comment:
@daniloprates please remove this console statement.

Reply:
@daniloprates did you remove it?

Reply:
yeap, https://github.com/input-output-hk/daedalus/commit/3179f1ff2688c4bd3ee74b6ffa6cbd2c0977828c

Reply:
uh sorry I have somehow missed it - GH didn't show it to me :(